chall06/llahti.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+#Python 3.8.2
+import sys
+import os
+
+def errorString(path, error):
+    format = "{}: {}: {}"
+    return format.format(sys.argv[0], path, error)
```

Comment:
I think it was not useful to create your own errorString function.

I also suggest to read the PEP8 python: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

It recommend snake_case function and variable names